contrib/jenkins_tests/rpm.sh

```diff
@@ -75,7 +76,7 @@ if [ $opt_binrpm -eq 1 ]; then
     if [ $opt_rpm -eq 1 ]; then
         test_exec="rpmbuild -bb $rpmmacros $rpmopts $rpmspec"
     else
-        test_exec="dpkg-buildpackage -us -uc -b"
+        test_exec="dpkg-buildpackage -us -uc -b -j8"
```

Comment:
please consider using ${make_opt} instead of -j8

Reply:
fixed